RELEASE.md

```diff
@@ -80,7 +80,13 @@ To publish a stable release:
 4. Wait until CI pipeline succeeded (once a tag is created, the release process through CircleCI will be triggered for this tag)
 5. Create a release in GitHub
    - Write the release notes (including a copy-paste of the changelog)
-   - Build binaries with `make disk` and attach them to the release
+   - Build binaries with `make dist` and attach them to the release
+6. Merge the release branch `release-x.y` to `master`
+   - Merge to `master` in the local checkout
+   - Fix any conflict and `git commit -s`
+   - Temporarily disable "Include administrators" in the [`master` branch protection rule](https://github.com/cortexproject/cortex/settings/branch_protection_rules)
+   - `git push`
```

Comment:
This is correct, if you know what you're doing. In my setup, it would push to incorrect fork though. (My `master` branch tracks my own fork).

I would perhaps be less specific with commands, and just say "push merge commit to Cortex master" – make sure to not push anything else.

Maybe I would instead say that it's best to do this work on a separate branch, not "master". Eg. `git checkout -b upstream_master upstream/master` (provided that your `upstream` remote is original fork, as suggested by GitHub own CLI tools), then do the merge, and then push back to `upstream`.

Reply:
I see your point. I guess we all have a different workflow and we can probably give `git` knowledge for granted and just mention "Push changes to upstream".

What's your take?

Reply:
I think that's fair assumption.

Reply:
I'd add "Double check before pushing!" :)

Reply:
Makes sense. What's about now?

Reply:
Looks good.